src/run/cluster.cpp

```diff
@@ -16,6 +16,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ****/
 
```

Comment:
This file should probably be moved to the /cluster/ directory.

Reply:
Done